com.holypangolin.Animatch.yml

```diff
@@ -0,0 +1,47 @@
+id: com.holypangolin.Animatch
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: animatch
+rename-icon: animatch
+rename-desktop-file: animatch.desktop
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=ipc
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.a
+  - /lib/*.la
+  - /share/aclocal
+  - /share/doc
+  - /bin/test_physfs
+modules:
+  - name: allegro5
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DALLEGRO_SDL=ON
+      - -DGL_BUILD_TYPE=gles2+
+      - -DWANT_DEMO=OFF
+      - -DWANT_EXAMPLES=OFF
+      - -DWANT_TESTS=OFF
+    sources:
+      - type: archive
+        url: https://github.com/liballeg/allegro5/releases/download/5.2.9.1/allegro-5.2.9.1.tar.gz
+        sha256: 0ee3fc22ae74601ad36c70afd793ff062f0f7187eeb6e78f8a24e5bf69170d30
+      - type: patch
+        path: stop-text-input.patch
+  - name: animatch
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://holypangolin.com/stuff/animatch/animatch-1.0.3.tar.gz
+        sha256: cea45539385007e240960d30694b0499778718d515eeb71bd23fe6f560db5398
+      - type: file
+        path: com.holypangolin.Animatch.appdata.xml
```

Comment:
this is never used.

Reply:
It overwrites the outdated file in the source package and then gets installed as usual.

Reply:
then it should be a patch.